src/world/message_server.cpp

```diff
@@ -22,10 +22,12 @@ along with this program.  If not, see http://www.gnu.org/licenses/
 #include <concurrentqueue.h>
 #include <memory>
 #include <queue>
+#include <set>
```

Comment:
Presumably this is already in the pch too yeah?

Reply:
I was having compile time issues without the include :S